apps/web/lib/supabase/realtime-disputes.ts

```
@@ -0,0 +1,20 @@
+import { createClient } from './server';
+
+// This utility sets up a subscription to the escrow_reviews table for real-time dispute updates.
+// Usage: call subscribeToDisputeUpdates(callback) on the client side.
+
+export function subscribeToDisputeUpdates(callback: (payload: any) => void) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)